docs/user_guide.md

```diff
@@ -43,32 +43,99 @@ See the [Docker documentation](../mcp-server-troubleshoot/DOCKER.md) for more de
 
 #### Using with MCP Clients
```

Comment:
This document and the Docker.md are duplicative. Pick one place to document this and remove duplication.